File: RadioJavan/Classes/Models/RadioJavanLogin.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Text.Json.Serialization;
using System.Threading.Tasks;

namespace RadioJavan.Classes.Models
{
    public class RadioJavanLogin
    {
        [JsonPropertyName("success")]
        public bool Success { get; set; }
        [JsonPropertyName("redirect")]
        public string Redirect { get; set; }
        [JsonPropertyName("user_id")]
        public int UserId { get; set; }
        [JsonPropertyName("msg")]
        public string Message { get; set; }
    }
}

File: Dart/lib/src/Classes/Models/RadioJavanLogin.dart
class RadioJavanLogin {
  bool Success;
  String Redirect;
  String UserId;
  String Message;

  RadioJavanLogin.fromJson(Map<String, dynamic> json)
      : Success = json['success'],
        Redirect = json['redirect'],
        UserId = json['user_id'],
        Message = json['msg'];
}
